tcms/testplans/tests/tests.py

```diff
@@ -148,6 +148,12 @@ def test_add_cases_sortkey_autoincrement(self):
         case_plan = plan.add_case(TestCaseFactory(), sortkey=15)
         self.assertEqual(15, case_plan.sortkey)
 
+    def test_get_full_url(self):
+        test_plan_url = self.plan_1.get_full_url()
```

Comment:
The previous implementation tried to verify that the last character was not a slash but didn't do it properly. Add an assert for that b/c the last character is the important thing for us here.